### DISCUSSION TITLE
Kibana can't populate log and Dashboard

### DISCUSSION THREAD

**Discussion by subas-007** [Security-Onion-Solutions/securityonion#1562] [Unsupported Versions]
<img width="866" alt="Untitled" src="https://user-images.githubusercontent.com/73147630/96528915-8e38fa00-12a3-11eb-93fe-f46db5fd37a9.png">
We have installed Security Onion 2.3 by replacing the old version, now Kibana is not showing any log however log size is increasing, did we missed anything here.
This is a fresh install from security onion iso file. And it is a standalone installation.
So-status shows all services running
I am very new to this community, Please suggest.

**Comment by weslambert** [comment]
Is this just for dashboards, or for the entire application?  Have you tried checking the log in /opt/so/log/kibana/kibana.log?
